lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -607,7 +692,7 @@ const std::string Fp16_to_Fp8E4M3B15x4 =
 // more than a single NaN values.
 
 #ifdef USE_ROCM
-static Value convert_val_Fp8E4M3FNUZ_to_Fp16(
+static Value Fp8E4M3FNUZ_to_Fp16_oneValue(
```

Comment:
Why did we convert one fp8 value at a time? I thought it would be more efficient to convert two fp8 values at once as it might use fewer instructions than calling this function twice?

Reply:
The reason we process one value each time is like:
v0 has e 0, while v1 has e with non-0, then the processing of v0 and v1 will be different which cannot be processed together unless we only handle the normal values.

Reply:
Alternatively, we can only handle the normal values, then we can process two values together. I am fine with either option. Any opinion?

Reply:
> Alternatively, we can only handle the normal values, then we can process two values together. I am fine with either option. Any opinion?

As it is MSFT that requested us to keep the SW based conversion, let's ask them which option that they would like (with the trade-off between accuracy and performance).

Reply:
And we can probably ask them and address in another PR. This PR does not need to wait for that.